src/Framework/PaymentGateways/Log/PaymentGatewayLog.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Give\Framework\PaymentGateways\Log;
+
+use Give\Log\Log;
+
+/**
+ * @unreleased
+ */
+class PaymentGatewayLog extends Log
+{
+    /**
+     * @inheritDoc
+     */
+    public static function __callStatic($name, $arguments)
+    {
+        $arguments[1]['category'] = 'Payment Gateway';
+        $arguments[1]['source'] = 'Payment Gateway';
+
+        if (defined('WP_DEBUG_LOG') && WP_DEBUG_LOG) {
+            parent::__callStatic($name, $arguments);
+        }
```

Comment:
Remove the wrapped if, here. We'll handle this elsewhere.

Reply:
resolved 506287cf